src/python/fuzzing/coverage_uploader.py

```diff
@@ -18,18 +18,18 @@
 from base import utils
 from bot.fuzzers import builtin_fuzzers
 from config import local_config
+from fuzzing import testcase_manager
 from google_cloud_utils import gsutil
 from google_cloud_utils import storage
 from metrics import logs
 from system import environment
-from system import shell
 
 LIST_FILE_BASENAME = 'file_list.txt'
-MAX_TESTCASE_DIRECTORY_SIZE = 1 * 1000 * 1000 * 1000  # 1 GB
-TESTCASES_PER_DAY = 1000
+TESTCASES_PER_DAY = 5000
```

Comment:
Let's change it to `2000` for now, or even leave `1000`? `gs://clusterfuzz-fuzzer-testcases/2019-06-17/` for example takes 35GB. I'm checking the yesterday's directory right now, as it may be even bigger.

Reply:
Ok putting back to 1000.